tests/data_files/keyfile_2048

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Can we have entries in the `Makefile` for all those keys? While so far we only have entries for certificates, I think it's mainly because we didn't have any need to generate new keys since the `Makefile` was introduced, and it would be good to have a section in the `Makefile` for keys too - imagine in a few years we need the same tests for 8192-bit RSA...

Reply:
@mpg Yes, I thought about that but didn't pursue it after I saw that only certificates are considered in the `Makefile`. If we're ok with providing information on key generation for _some_ (new) keys only, I can add the corresponding instructions to the `Makefile`.

Reply:
I think more (executable) documentation can never hurt, so I don't really see a reason for documenting only certificate generation and not key generation in `data_files/Makefile`.

Also, I think so far we only document new things that way (old certs are not in the Makefile either), so that wouldn't be any different for keys and certificates. Of course we could go back and re-generate all existing keys and certs but that would be a lot of work, and I don't think skipping that work should prevent us from documenting new things better.

@gilles-peskine-arm would you agree?

Reply:
@mpg I agree - I'll add the commands for the new keys to the Makefile.

Reply:
Indeed the reason `Makefile` only contains rules to generate certificates is that it only documents new things, and the new things have only included certificates so far. My intent is that everything added to `data_files` should be documented via the makefile, and we should play catch-up with already-existing data When We Have Time™.